cmd/repl/repl.go

```diff
@@ -177,22 +178,30 @@ func runStmt(stmt string, isTerminal bool, modelDir string, ds string) error {
 	tableRendered := false
 	table := tablewriter.NewWriter(os.Stdout)
 	sess := makeSessionFromEnv()
-	if ds != "" {
-		sess.DbConnStr = ds
-	}
-
-	stream := sql.RunSQLProgram(stmt, modelDir, sess)
-	for rsp := range stream.ReadAll() {
-		// pagination. avoid exceed memory
-		if render(rsp, table, isTerminal) && table.NumLines() == tablePageSize {
+	sess.DbConnStr = getDataSource(ds, currentDB)
+	statements := strings.Split(stmt, ";")
```

Comment:
I don't get the point here. The parameter stmt seems a SQL statement. Why do we need to split it into multiple statements?

Even if we need to, `strings.Split(stmt, ";")` doesn't work, as the character `;` might appear in a string constant. For example,

```sql
SELECT * FROM tbl WHERE a==";";
```

Reply:
> I don't get the point here. The parameter stmt seems a SQL statement. Why do we need to split it into multiple statements?
> 
> Even if we need to, `strings.Split(stmt, ";")` doesn't work, as the character `;` might appear in a string constant. For example,
> 
> ```sql
> SELECT * FROM tbl WHERE a==";";
> ```

This is an existing problem because `readStmt`  assumes SQL statements are separated by substrings semicolon :
https://github.com/sql-machine-learning/sqlflow/blob/develop/cmd/repl/repl.go#L53-L55

After this change, if the user inputs 
```sql
USE iris; SELECT * FROM table;
```
The statements will work as expected.

I'll find a way to recognize quoted strings